src/components/Divider.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+const Divider = ({color, style, ...other}) => (
+  <hr style={{borderTopColor: color || 'rgba(0, 0, 0, 0.12)', ...style}} />
```

Comment:
Nice! As long as it looks good 😎 

Maybe just remove the `()` for maximum minimalism?